arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -74,14 +74,7 @@
         "operationId": "Servers_Create",
         "parameters": [
           {
-            "name": "serverParameters",
-            "in": "body",
-            "required": true,
-            "schema": { "$ref": "#/definitions/AnalysisServicesResource" },
-            "description": "Request body for provisioning"
-          },
-          {
-            "name": "analysisServerName",
+            "name": "serverName",
```

Comment:
looking at the sequence, I have usually seen resourcegroupName as the first parameter, then the RP specific artifact name, any query parameters and then the body parameter. If would be nice if you can follow the same sequence.
This sequence translates to the method signature.